docs/models.rst

```diff
@@ -76,14 +72,14 @@ function.  However the foreign key has to be separately declared with the
 
     class Person(db.Model):
         id = db.Column(db.Integer, primary_key=True)
-        name = db.Column(db.String(50))
-        addresses = db.relationship('Address', backref='person',
-                                    lazy='dynamic')
+        name = db.Column(db.String(50), nullable=False)
+        addresses = db.relationship('Address', backref='person', lazy=True)
```

Comment:
Don't need `lazy=True`, that's the default. (Well, `'select'` is the default, `True` is a synonym.)